spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper do
+  describe '#telephone_number' do
+    it 'wraps the phone number in appropriate markup' do
+      expect(helper.telephone_number('01234 567 890'))
+        .to eq(%(<span itemprop="telephone"><a href="tel:01234567890">01234567890</a></span>))
```

Comment:
Good point! Will investigate

Reply:
Ok, that was weird: I was (incorrectly) using `gsub!` instead of `gsub`, and I guess that gets evaluated first because in the markup href comes before the text.

Damn mutation!